requirements-dev.txt

```diff
@@ -3,5 +3,5 @@ pytest>=5.2     # due to pytest-flask, pytest-cov
 mock
 pytest-mock
 snapshottest
-
+deprecated
```

Comment:
Did we have a dependency called deprecated or what is this?

Reply:
I removed the usage of the deprecated method and the usage of this library